extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -424,6 +430,54 @@ export class AmpAnalytics extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Returns a promise that resolves when configuration is re-written if
+   * configRewriter is configured by a vendor.
+   *
+   * @private
+   * @return {!Promise<undefined>}
+   */
+  rewriteConfig_() {
+    const remoteConfigUrl = this.getConfigRewriter_()['url'];
+
+    const config = dict({});
+    const inlineConfig = this.getInlineConfigNoInline();
+    this.validateTransport_(inlineConfig);
+    this.mergeObjects_(inlineConfig, config);
+    this.mergeObjects_(this.remoteConfig_, config);
+
+    if (!remoteConfigUrl || this.isSandbox_) {
+      this.rewrittenConfig_ = config;
+      // use default configuration merge.
+      return Promise.resolve();
+    }
+
+    assertHttpsUrl(remoteConfigUrl, this.element);
+    const TAG = this.getName_();
+    dev().fine(TAG, 'Rewriting config', remoteConfigUrl);
+
+    const fetchConfig = {
+      method: 'POST',
+      body: config,
+      credentials: 'include',
```

Comment:
let's read it from attribute `data-credentials`, like what remote config is doing.

Reply:
Done